cpp/src/parquet/arrow/writer.cc

```diff
@@ -63,6 +66,10 @@ using arrow::Status;
 using arrow::Table;
 using arrow::TimeUnit;
 
+using ArrowTypeId = arrow::Type;
```

Comment:
nit: this actually makes the code less understandable to me and it seems like it only saves 1 character?  (not a big deal either way though).

Reply:
I can revert :-)